src/CoreWCF.ConfigurationManager/tests/IntegrationOfDependencyInjectionTests.cs

```diff
@@ -21,18 +21,15 @@ public IntegrationOfDependencyInjectionTests(ITestOutputHelper output)
         [Fact]
         public void AddOneEndpointInContainer()
         {
-            IWebHost host = ServiceHelper.CreateWebHostBuilder<Startup>(_output).Build();
-            using (host)
-            {
-                host.Start();
-                var resolver = new DependencyResolverHelper(host);
-                IServiceBuilder serviceBuilder = resolver.GetService<IServiceBuilder>();
+            using IWebHost host = ServiceHelper.CreateWebHostBuilder<Startup>(_output).Build();
```

Comment:
As you won't be modifying the UseNetTcp method, this test will need to be reverted. You could add a new test which explicitly passes `IPAddress.IPv6Any` to CoreWCF to make sure it continues to work in the future with the base address fix.

Reply:
Test added